### PR TITLE
Changes to the manual

### PR DESCRIPTION
**Pull request by dvandyk**
- clean up old references to ```eosdata```
 - add reference to the Amoroso paper
 - add 'How to add a new constraint' to the manual

@greenwald Once you accept the invitation I can assign you as a reviewer.

### CODE REVIEW

File: manual/doc/extending.tex [hunk 1]
```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
...
+    \item \texttt{Gaussian},
+    \item \texttt{MultivariateGaussian},
+    \item \texttt{MultivariateGaussian(Covariance)}.
+\end{itemize}
+
+\subsection{Type \texttt{Amoroso}}
+
+Type \texttt{Amoroso} uses a likelihood arising from the \gls{PDF} of the
+Amoroso distribution \cite{Crooks:2010}, a four-parameter distribution. This
+distribution is useful for the modelling of (sequences) of upper limits on
```
Comment (cBobeth) [inline manual/doc/extending.tex]: "... of (sequences) of upper limits ..."
 maybe to
"... (of sequences) of upper limits ..."

File: manual/doc/extending.tex [hunk 2]
```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
...
+as-of-yet undiscovered decays at given probabilities. For example, the upper
+limits on the decay $B_s \to \mu^+\mu^-$ by the LHCb experiment prior to the
+discovery of this decay are modelled using the \texttt{Amoroso} type of
```
Comment (cBobeth) [inline manual/doc/extending.tex]: "... are modelled using ..."
maybe 
"... were modelled using ..."

You could give also the "qualified" name for this constraint

File: manual/doc/extending.tex [hunk 3]
```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
...
+likelihood. The following keys are required in the description of the
+constraint:
+\begin{description}[labelwidth=.15\textwidth]
+    \item[\texttt{observable}] the qualified name of an \texttt{Observable} or a \texttt{Parameter};
+    \item[\texttt{kinematics}] the map representation of the kinematic variables pertaining to the observable;
+    \item[\texttt{options}] the map represenation of the options pertaining to the observable;
+    \item[\texttt{mean}] the mean of the distribution;
+    \item[\texttt{sigma-stat}] the map representation with keys \texttt{hi} and \texttt{lo} for the
+    upper and lower statistical uncertainty of the distribution, respectively;
+    \item[\texttt{sigma-sys}] the same as \texttt{sigma-stat}, but for the systematic uncertainty;
```
Comment (cBobeth) [inline manual/doc/extending.tex]: Maybe write how are 'sigma-stat' and 'sigma-sys' combined?

File: manual/doc/extending.tex [hunk 4]
```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
...
+    \item[\texttt{dof}] the degrees of freedom associated with this observation (should default to $1$).
```
Comment (cBobeth) [inline manual/doc/extending.tex]: Maybe give a comment for what a non-default value could be useful. As far as I remember putting a zero would exclude this constraint from being considered a degree of freedom in the p-value calculation.

File: manual/doc/extending.tex [hunk 5]
```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
...
+\end{description}
+
+\subsection{Type \texttt{MultivariateGaussian}}
+
+Type \texttt{MultivariateGaussian} uses a multivariate Gaussian likelihood,
+parametrised in terms of its mode and correlation matrix. The following keys
+are required in the description of the constraint:
+\begin{description}[labelwidth=.15\textwidth]
+    \item[\texttt{dim}] the dimensionality of the multivariate distribution;
+    \item[\texttt{observable}] the list of size \texttt{dim} of qualified names for for either \texttt{Observable}s or \texttt{Parameter}s;
```
Comment (cBobeth) [inline manual/doc/extending.tex]: "... for for ..." to "... for ..."

File: manual/doc/extending.tex [hunk 6]
```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
...
+    \item[\texttt{kinematics}] the list of size \texttt{dim} of map representations of the kinematic variables pertaining to each observable;
+    \item[\texttt{options}] the list of size \texttt{dim} of map represenations of the options pertaining to each observable;
```
Comment (cBobeth) [inline manual/doc/extending.tex]: typo: 'representations'

File: manual/doc/extending.tex [hunk 7]
```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
...
+    \item[\texttt{mean}] the list of size \texttt{dim} describing the mean of the distribution;
+    \item[\texttt{sigma-stat-hi}] the list of size \texttt{dim} for the upper statistical uncertainty of the distribution;
+    \item[\texttt{sigma-stat-lo}] the list of size \texttt{dim} for the lower statistical uncertainty of the distribution;
+    \item[\texttt{correlations}] the matrix of size \texttt{dim} $\times$ \texttt{dim} for the statistical correlations;
+    \item[\texttt{sigma-sys}] the list of size \texttt{dim} for the systematic uncertainty;
```
Comment (cBobeth) [inline manual/doc/extending.tex]: Describe how 'sigma-sys' is combined with the other 'sigma-stat-hi', 'sigma-stat-lo' and 'correlations'.

File: manual/doc/extending.tex [hunk 8]
```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
...
+\subsection{Type \texttt{MultivariateGaussian(Covariance)}}
+
+Type \texttt{MultivariateGaussian(Covariance)} uses a multivariate Gaussian
+likelihood, parametrised in terms of its mode and covariance matrix. The
+following keys are required in the description of the constraint:
+\begin{description}[labelwidth=.15\textwidth]
+    \item[\texttt{dim}] the dimensionality of the multivariate distribution;
+    \item[\texttt{observable}] the list of size \texttt{dim} of qualified names for for either \texttt{Observable}s or \texttt{Parameter}s;
+    \item[\texttt{kinematics}] the list of size \texttt{dim} of map representations of the kinematic variables pertaining to each observable;
+    \item[\texttt{options}] the list of size \texttt{dim} of map represenations of the options pertaining to each observable;
```
Comment (cBobeth) [inline manual/doc/extending.tex]: typo: representation

File: manual/examples/x-constraints.yaml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+X::mass+decay-constant@Example:
+    type: MultvariateGaussian
```
Comment (cBobeth) [inline manual/examples/x-constraints.yaml]: typo: Multivariate

File: manual/doc/extending.tex (hunk 1 above)
Comment (dvandyk) [reply manual/doc/extending.tex]: Rephrased.

File: manual/doc/extending.tex (hunk 2 above)
Comment (dvandyk) [reply manual/doc/extending.tex]: I would say, we stick to present tense as in a paper. Also, they still *are* modelled this way.

File: manual/doc/extending.tex (hunk 3 above)
Comment (dvandyk) [reply manual/doc/extending.tex]: Added:
```
By default the uncertainties are symmetrised, and statistical und systematical
uncertainties are added in quadrature.
```

File: manual/doc/extending.tex (hunk 4 above)
Comment (dvandyk) [reply manual/doc/extending.tex]: I think I want to get rid of this anyway. So it makes no sense to document it right now.

File: manual/doc/extending.tex (hunk 5 above)
Comment (dvandyk) [reply manual/doc/extending.tex]: Thanks, fixed.

File: manual/doc/extending.tex (hunk 6 above)
Comment (dvandyk) [reply manual/doc/extending.tex]: Fixed all four of them.

File: manual/doc/extending.tex (hunk 7 above)
Comment (dvandyk) [reply manual/doc/extending.tex]: Done.

**Comment by dvandyk** [comment]
@greenwald I will merge this today already, since I need it for the 0.2.0 release today. The latter is required for interoperability with ```wcxf```, specifically for the resubmission to the journal.

**Comment by greenwald** [comment]
@dvandyk, sorry---was traveling to Charm'18 and missed the notifications. When you return to editing, let me know.

**Pull request merged**